scripts/run_olserver.sh

```diff
@@ -8,17 +8,18 @@ cd /opt/openlibrary
 git pull origin master
 cd /opt/openlibrary/vendor/infogami
 git pull origin master
+if [[ -d "/opt/booklending_utils" ]] ; then
+    cd /opt/booklending_utils
+    git pull origin master
+fi
```

Comment:
Let's simplify our lives and assume it exists.

```suggestion
cd /opt/booklending_utils
git pull origin master
```